### ISSUE TITLE
Corporate words

### ISSUE THREAD

**Issue by johnbent** [inclusivelint/inclusivelint-lib#22]
Hello.  Thank you for this project!  It appears that the scan here uses this wordlist: [https://github.com/inclusivelint/inclusivelint-lib/blob/main/src/wordsTable.md.](https://github.com/inclusivelint/inclusivelint-lib/blob/main/src/wordsTable.md.)  Is it possible to pass an argument for an additional word list.  For example, in our project (https://github.com/Seagate/cortx), we would very much like to add something like inclusivelint and scan our code for problematic language as in your wordlist.  However, we also want to scan our code for problematic language that is private to Seagate such as internal unannounced product names.  So, instead of having a separate scan for different language, we would like to just do a single scan which checks both words found in a shared public list (like yours) as well as checking for words that are private to us.

So basically would it be possible to add an argument to inclusivelint to add an additional list of words to be scanned?
